### PR TITLE
fix: don't hide header at any viewport size

### PR DESCRIPTION
**Pull request by vpicone**
@mjabbink This brings the header more in line with the shell guidance:

>The header spans the full width of the viewport and is the topmost element in the browser window. Header elements are persistent throughout the product experience.

[source](https://www.carbondesignsystem.com/components/UI-shell-header/usage#anatomy)

It also pulls the page tabs out of the header so their sticky logic doesn't depend on page header height.

### CODE REVIEW

File: packages/gatsby-theme-carbon/src/components/PageTabs/PageTabs.module.scss [hunk 1]
```diff
@@ -1,7 +1,10 @@
 .tabs-container {
   margin: 0;
   width: 100%;
-  box-shadow: 0 -1px 0 rgba($ui-02, 0.25);
```
Comment (alisonjoseph) [inline packages/gatsby-theme-carbon/src/components/PageTabs/PageTabs.module.scss]: Is this removed on purpose??

File: packages/gatsby-theme-carbon/src/components/PageHeader/PageHeader.module.scss [hunk 2]
```diff
@@ -1,28 +1,19 @@
...
 
 .page-header {
-  background: $carbon--black-100;
+  background: $ui-05;
   color: $inverse-01;
   width: 100%;
   display: flex;
   flex-direction: column;
   justify-content: flex-end;
-  height: 20rem;
```
Comment (alisonjoseph) [inline packages/gatsby-theme-carbon/src/components/PageHeader/PageHeader.module.scss]: is this height change on purpose?

File: packages/gatsby-theme-carbon/src/components/PageTabs/PageTabs.module.scss (hunk 1 above)
Comment (vpicone) [reply packages/gatsby-theme-carbon/src/components/PageTabs/PageTabs.module.scss]: Yeah it was emulating a border top which is now a proper border bottom in the page header.

File: packages/gatsby-theme-carbon/src/components/PageHeader/PageHeader.module.scss (hunk 2 above)
Comment (vpicone) [reply packages/gatsby-theme-carbon/src/components/PageHeader/PageHeader.module.scss]: Yeah the tabs are removed

Comment (alisonjoseph) [reply packages/gatsby-theme-carbon/src/components/PageHeader/PageHeader.module.scss]: It's supposed to be the same height with or without tavs, the 20rem is correct. Unless there is an updated design somewhere??

File: packages/gatsby-theme-carbon/src/components/PageHeader/PageHeader.module.scss [hunk 3]
```diff
@@ -1,28 +1,19 @@
 @import '~carbon-components/scss/globals/scss/layout';
 
 .page-header {
-  background: $carbon--black-100;
+  background: $ui-05;
```
Comment (alisonjoseph) [inline packages/gatsby-theme-carbon/src/components/PageHeader/PageHeader.module.scss]: Is there a reason we changed this from black to gray-100?

File: packages/gatsby-theme-carbon/src/components/PageTabs/PageTabs.module.scss (hunk 1 above)
Comment (alisonjoseph) [reply packages/gatsby-theme-carbon/src/components/PageTabs/PageTabs.module.scss]: 👍

File: packages/gatsby-theme-carbon/src/components/PageTabs/PageTabs.module.scss [hunk 4]
```diff
@@ -1,7 +1,10 @@
...
+  background: $ui-05;
```
Comment (alisonjoseph) [inline packages/gatsby-theme-carbon/src/components/PageTabs/PageTabs.module.scss]: Same as the above question, should be black not gray-100?

**Comment by alisonjoseph** [comment]
Overall looks good, had a few comments about some design changes in this PR though, not sure where they are coming from??

File: packages/gatsby-theme-carbon/src/components/PageHeader/PageHeader.module.scss (hunk 2 above)
Comment (vpicone) [reply packages/gatsby-theme-carbon/src/components/PageHeader/PageHeader.module.scss]: Ah, unintentional. To change the height now, you'll just need to set the height custom property. It'll shorten it by 3rem if there's tabs.

```css
--height: 14rem
```

File: packages/gatsby-theme-carbon/src/components/PageHeader/PageHeader.module.scss (hunk 3 above)
Comment (vpicone) [reply packages/gatsby-theme-carbon/src/components/PageHeader/PageHeader.module.scss]: Whoops, was trying to switch to use theme tokens and misread this.

File: packages/gatsby-theme-carbon/src/components/PageTabs/PageTabs.module.scss (hunk 4 above)
Comment (vpicone) [reply packages/gatsby-theme-carbon/src/components/PageTabs/PageTabs.module.scss]: same, switched it back

**Pull request merged**